charmcraft.yaml

```diff
@@ -1,13 +1,25 @@
 # Copyright 2024 Canonical Ltd.
 # See LICENSE file for licensing details.
-# This file configures Charmcraft.
-# See https://juju.is/docs/sdk/charmcraft-config for guidance.
-
 type: charm
 bases:
   - build-on:
     - name: ubuntu
       channel: "22.04"
+      architectures: [amd64]
```

Comment:
I'm wondering if there is a need for the square brackets here :) `[]`

Reply:
I'll still need to decide whether we should support multiple archs here, and also there's this point of moving to charmcraft v3 which will change this completely, so I think we can leave it like this for now